rules.js

```diff
@@ -28,7 +28,8 @@ export default [
   [/^ring(?:-(.*))$/, ([_]) => emitWarning(_, TYPES.removed)],
   // filters
   [/^(backdrop-)?(filter|filter-none)$/, ([_]) => emitWarning(_, TYPES.removed)],
-  [/^(backdrop-)?blur(-(.*))?$/, ([_]) => emitWarning(_, TYPES.removed)],
+  [/^blur(-(.*))?$/, ([_]) => emitWarning(_, TYPES.removed)],
```

Comment:
Was it intended to also match `blur`?

Reply:
At least the rule before said that the "backdrop-" prefix was optional. 🤷 😄 So I kept it as it was and only removed the warning for `backdrop-blur`.

Reply:
Fair enough :D